src/components/form/FormProgressStepper.tsx

```diff
@@ -0,0 +1,121 @@
+import { CheckCircleOutline } from "@mui/icons-material";
+import { Box, Theme, styled } from "@mui/material";
+import { Typography } from "elements";
+import theme from "theme";
+
+interface StepBoxProps {
+  readonly boxColor: keyof Theme["colors"];
+  readonly boxTitle: string;
+}
+
+interface FormStepperGridProps {
+  readonly activeStep: number;
+  readonly stepTitles: ReadonlyArray<string>;
+}
+
+const statusColor = (activeStep: number, currentIndex: number) => {
+  const isStepCompleted = activeStep > currentIndex + 1;
+  const isStepActive = activeStep === currentIndex + 1;
+
+  if (isStepCompleted) {
+    return "green";
+  } else if (isStepActive) {
+    return "music";
+  } else {
+    return "grey400";
+  }
+};
+
+const StepBox = styled(Box)<StepBoxProps>`
+  flex-grow: 1;
+  color: ${(props) => theme.colors[props.boxColor]};
+  background-color: ${theme.colors.black};
+  border-bottom: 3px solid ${(props) => theme.colors[props.boxColor]};
+  height: 59px;
+  width: 232px;
+  display: flex;
+  align-items: center;
+`;
```

Comment:
Just didn't think of going in that direction, as progress steppers are a bit different functionality/design wise. I didn't use the MUI progress stepper either because it didn't conform as well to the design look and feel.